langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/AbstractOllamaInfrastructureVisionModel.java

```diff
@@ -0,0 +1,105 @@
+package dev.langchain4j.model.ollama;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import com.github.dockerjava.api.model.Image;
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.LazyFuture;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j
+public class AbstractOllamaInfrastructureVisionModel {
```

Comment:
Looks like only `MODEL` is different comparing to existing `AbstractOllamaInfrastructure`, so maybe instead of copy-pasting it completely, we should introduce something like `protected abstract String model();` and let classes implement it

Reply:
seems ok for me

Reply:
i could not figure it how to add model name method easily. So many static things in `AbstractOllamaInfrastructure` dont easy things up. I will look at this later

Reply:
@bidek no worries, I planned to simplify this Ollama IT setup anyway, will do it in scope of it a bit later.